roles/neutron-common/templates/etc/neutron/dhcp_agent.ini

```diff
@@ -6,6 +6,8 @@ debug = {{ neutron.logging.debug }}
 
 enable_isolated_metadata = True
```

Comment:
according to docs `This option doesn't have any effect when force_metadata is set to True.`   do we need to flag it as well ?